ykrt/src/compile/jitc_yk/codegen/x64/lsregalloc.rs

```diff
@@ -269,6 +273,10 @@ impl LSRegAlloc<'_> {
         self.spills[usize::from(iidx)] = SpillState::Stack(frame_off);
     }
 
+    pub(crate) fn assign_const(&mut self, iidx: InstIdx, bits: u32, v: u64) {
```

Comment:
Add a doc string. [Should be easy to copy from the other similar functions.] I also think this should be `assign_const_int` because it can't (yet) handle other `Const` kinds.

Reply:
[0022632](https://github.com/ykjit/yk/pull/1422/commits/00226328b8a1cfff73af85e11612135268003639)